_post: RIP pullmoll (#139)
void-linux/void-linux.github.io: _posts/2021-03-27-RIP-pullmoll.markdown

---
layout: post
title: "RIP pullmoll"
comments: true
---

<center>
	<img src="assets/img/pullmoll.png">
</center>

> Today we were informed that our long time contributor and maintainer Jürgen Buchmüller (pullmoll) passed away.

Reviewer: ```suggestion
Today we were informed that our long time contributor and maintainer Jürgen Buchmüller (pullmoll) passed away.
```

He was not only a dedicated and driven developer in the Void Linux project, he also was one of the friendliest and warmest person we had the pleasure to work with. Jürgen will be missed as a steady force and a calm and wise voice in discussions. Most of all, he will be remembered as a friend to all of us.

> **Thank you pullmoll.**

Reviewer: ```suggestion
**Thank you, pullmoll.**
```

---
layout: post
title: "RIP pullmoll"

> comments: true

Reviewer: What does this do?

Also, remember to squash the commits together before merging.

---

<center>
	<img src="assets/img/pullmoll.png" alt="Picture of Pullmoll">
</center>

Today, we were informed that our long time contributor and maintainer Jürgen
Buchmüller (pullmoll) has passed away.

With boundless dedication, Jürgen was one of the most prolific contributors to

> the Void project and sponsored the project financially too.

Reviewer 2: ```
financially, too.
```

> the Void project and sponsored the project financially, too.

Reviewer 3: `s/Void /Void Linux/`, too.

Author: applied

> He investeded lots of his time in keeping gcc up to date, supplied tons of

Reviewer 2: `invested`

> patches to port packages to new compilers and platforms and also cared about

Reviewer 2: `s/cared about/took care of/`

> He invested lots of his time in keeping gcc up to date, supplied tons of
> patches to port packages to new compilers and platforms and also took care of
> simple updates of packages.

Reviewer: ```suggestion
He invested lots of his time in keeping GCC up to date and well tested, supplied tons of
patches to port packages to new compilers and platforms and also took care of
updating many packages, including several non-trivial ones, like OpenJDK and LibreOffice.
```

Author: This is much better, thank you.

He was not only a dedicated and driven developer in the Void Linux project, he

> also was one of the friendliest and warmest person we had the pleasure to work

Reviewer: people instead of person, suggested by ifreund

Author: applied

with. Jürgen will be missed as a steady force and a calm and wise voice in

> discussions. Most of all, he will be remembered as a friend to all of us.

Reviewer: I'd like to leave something to friends and family. Maybe it fits here?

> Our thoughts and condolences go out to Jürgen's family, friends, and loved ones.

Author: applied
